constants.xml

```diff
@@ -287,7 +287,7 @@
     </recovery>
     <smart_contract>
         <ENABLE_SC>true</ENABLE_SC>
-        <SCILLA_ROOT>/scilla</SCILLA_ROOT>
+        <SCILLA_ROOT>/home/nathan/repos/Zilliqa/scilla</SCILLA_ROOT>
```

Comment:
This is an error is assume, you didnt mean to change this in the master

Reply:
nope